identity/app/views/advertsManager.scala.html

```diff
@@ -6,17 +6,17 @@
 
 <div class="l-side-margins">
 
-<div class="identity-ad-prefs-area identity-ad-prefs-area--header">
-    <div class="gs-container">
-        <header class="identity-ad-prefs-area__section">
-            <h1 class="identity-title">Cookies and advertising settings</h1>
-        </header>
-    </div>
-</div>
-
 <div class="identity-ad-prefs-area">
 
     <div class="gs-container">
+         <section class="identity-ad-prefs-area__section">
+            <h2 class="identity-title">Your Privacy</h2>
```

Comment:
if the `h1` is gone could we turn these into `h1` then `h2`? I think that's preferable to not having an h1 but I might be wrong, @SiAdcock ?

Reply:
I do agree. Is there a technical reason we can't follow the normal header hierarchy @rcrphillips?

Reply:
Thanks both. Have amended that now.